Prettier v2.0.5 breaks linting

Nearly identical stacktrace to #441. Makes me think something is wrong in how XO uses Prettier.

```js
/home/runner/work/nextjs-starter/nextjs-starter/node_modules/.bin/xo
Error: Failed to load plugin 'prettier' declared in 'CLIOptions': Cannot find module 'eslint-plugin-prettier'
Require stack:
- /home/runner/work/nextjs-starter/nextjs-starter/__placeholder__.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.resolve (internal/modules/cjs/helpers.js:83:19)
    at Object.resolve (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/shared/relative-module-resolver.js:44:50)
    at ConfigArrayFactory._loadPlugin (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:959:39)
    at /home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:848:33
    at Array.reduce (<anonymous>)
    at ConfigArrayFactory._loadPlugins (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:844:22)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:667:32)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/home/runner/work/nextjs-starter/nextjs-starter/node_modules/eslint/lib/cli-engine/config-array-factory.js:596:20)
```

--- a/package.json
+++ b/package.json
@@ -86,7 +86,7 @@
 		"open-editor": "^2.0.1",
 		"p-reduce": "^2.1.0",
 		"path-exists": "^4.0.0",
-		"prettier": "2.0.4",
+		"prettier": "^2.1.2",
 		"resolve-cwd": "^3.0.0",
 		"resolve-from": "^5.0.0",
 		"semver": "^7.3.2",

